encoding/codecv3/codecv3.go

```diff
@@ -52,6 +57,11 @@ func NewDAChunk(chunk *encoding.Chunk, totalL1MessagePoppedBefore uint64) (*DACh
 	return codecv2.NewDAChunk(chunk, totalL1MessagePoppedBefore)
 }
 
+// DecodeDAChunksRawTx takes a byte slice and decodes it into a []DAChunkRawTx.
+func DecodeDAChunksRawTx(bytes [][]byte) ([]*codecv2.DAChunkRawTx, error) {
```

Comment:
```suggestion
// DecodeDAChunksRawTx takes a byte slice and decodes it into a []*DAChunkRawTx.
func DecodeDAChunksRawTx(bytes [][]byte) ([]*DAChunkRawTx, error) {
```

Reply:
or you can just remove `DecodeDAChunksRawTx` and `DecodeTxsFromBlob`, because by using `type DAChunkRawTx codecv2.DAChunkRawTx`, these functions can be reused.